local_build.sh

```diff
@@ -26,9 +26,9 @@ if [[ "$#" -eq "1" ]]; then
         MOUNT="/stackstate-agent-mount"
 
         docker run --rm \
-            -e ARTIFACTORY_USER=$ARTIFACTORY_USER \
-            -e ARTIFACTORY_PASSWORD=$ARTIFACTORY_PASSWORD \
-            -e ARTIFACTORY_URL="artifactory.stackstate.io/artifactory/api/pypi/pypi-local" \
+            -e artifactory_user=$artifactory_user \
+            -e artifactory_password=$artifactory_password \
```

Comment:
Would this not just assign the value of the ENV vars back to themselves?

Reply:
the docker container does not inherit the variables from your local, you need to explicitly set it